pkgs/development/python-modules/textx/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, arpeggio
+, click
+, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "textX";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname;
+    inherit version;
+    sha256 = "00mwd588ms96qp27m5vpjkzk30wfw53hnmv8y77slxca8lw9vq82";
+  };
+
+  nativeBuildInputs = [ pytestCheckHook ];
```

slightly more correct
```suggestion
  checkInputs = [ pytestCheckHook ];
```